Makefile

```diff
@@ -296,3 +296,14 @@ catalog-build: opm ## Build a catalog image.
 .PHONY: catalog-push
 catalog-push: ## Push a catalog image.
 	$(MAKE) docker-push IMG=$(CATALOG_IMG)
+
+# Variables required to run and build LVM end to end tests.
+LVM_OPERATOR_INSTALL ?= true
+LVM_CLUSTER_UNINSTALL ?= true
```

Comment:
- maybe, have these options read similar, like `LVM_OPERATOR_(INSTALL|UNINSTALL)` or `LVM_CLUSTER_(INSTALL|UNINSTALL)`

Reply:
Agree. 
I think the arguments should only control whether the operator should be installed/uninstalled. The LVMCluster will always be created/deleted. This would help if we want to rerun the tests on the same operator image.

Reply:
should I rename these?

Reply:
So, does that mean that we dont need lvm_cluster_install argument to be passed, right?

Reply:
it will be replaced by lvm_operator_uninstall and the after_suite function will be updated accordingly